lib/puppet/provider/foreman_smartproxy/rest_v2.rb

```diff
@@ -2,6 +2,12 @@
 
   confine :feature => :apipie_bindings
 
+  def raise_error(e)
+    body = JSON.parse(e.response)["error"]["full_messages"].join(" ") rescue 'N/A'
+    raise("Error while registering proxy (#{e.message}): #{body}, " +
+      "registration can be skipped by unsetting --register-in-foreman option.")
```

Comment:
Re: name of parameter - I don't see any related comment on that.

Can you suggest wording you expect? I am really bad in this... I don't follow what debug output from bindings you mean.